receiver/purefareceiver/receiver.go

```diff
@@ -50,19 +50,18 @@ func (r *purefaReceiver) Start(ctx context.Context, compHost component.Host) err
 	scrapeCfgs := []*config.ScrapeConfig{}
 
 	commomLabel := model.LabelSet{
-		"env":           model.LabelValue(r.cfg.Env),
-		"fa_array_name": model.LabelValue(r.cfg.Endpoint),
-		"host":          model.LabelValue(r.cfg.Endpoint),
+		"deployment.environment": model.LabelValue(r.cfg.Env),
+		"host.name":              model.LabelValue(r.cfg.Endpoint),
```

Comment:
`host.name` should have the same value as `arr.Address` on the endpoints /metrics/array, /metrics/directories, /metrics/pods. 

the label `host` from the metrics from /metrics/hosts, and /metrics/volumes should be re-labeled as `host.name` and should overwrite the `host.name` that comes from the otel-col auto-discovery

Reply:
Hmm. Yes, I think that I've got. Thanks for this review

cc @jpkrohling

Reply:
cool. let me build and test, before giving my full approval. Will do first thing tomorrow morning. Thanks!